packages/imagemin/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@neutrinojs/imagemin",
+  "version": "7.3.2",
+  "description": "Neutrino middleware for optimizing images with imagemin",
+  "main": "index.js",
+  "keywords": [
+    "neutrino",
+    "neutrino-middleware",
+    "image",
+    "imagemin"
+  ],
+  "author": "Tim Kelty <timkelty@gmail.com>",
+  "license": "MPL-2.0",
+  "repository": "https://github.com/mozilla-neutrino/neutrino-dev/tree/master/packages/imagemin",
```

Comment:
Does this repo format actually work on npm?

Reply:
Hmmm not sure - I was just following suit with the rest of the middlewares: https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/web/package.json#L14

Reply:
wtf. When I was viewing this diff, it showed a different format...don't mind me.